### PR TITLE
Update Readmes

### PR DESCRIPTION
**Pull request by Nitwel**

### CODE REVIEW

File: src/components/v-form/readme.md [hunk 1]
```diff
@@ -13,19 +13,20 @@ Renders a form using interfaces based on the passed collection name.
...
 n/a
 
 ## Events
-| Event          | Description                                                              |
-|----------------|--------------------------------------------------------------------------|
-| `update:edits` | Update the edits state. Enables the `.sync` modifier on the `edits` prop |
+| Event   | Description | Value |
+|---------|-------------|-------|
+| `input` |             |       |
+<!-- | `update:edits` | Update the edits state. Enables the `.sync` modifier on the `edits` prop |       | -->
```
Comment (rijkvanzanten) [inline src/components/v-form/readme.md]: what's this?

File: src/components/v-item-group/readme.md [hunk 2]
```diff
@@ -175,3 +175,21 @@ n/a
 
 ### Item (`v-item`)
 n/a
+
```
Comment (rijkvanzanten) [inline src/components/v-item-group/readme.md]: Ah this file is an interesting edge case. Currently it has the tables for all the subcomponents too.

Lets make readme's for all individual children instead so we can prevent this

**Comment by rijkvanzanten** [comment]
Do you think you'll have a chance to tweak the build process a notch @nitwel? 🙂

**Comment by Nitwel** [comment]
Heyy, sure. I'm home again on saturday, then i'll have a look at it.

File: src/components/v-item-group/readme.md (hunk 2 above)
Comment (Nitwel) [reply src/components/v-item-group/readme.md:178]: Does storybook understand this tho?

File: src/components/v-button-group/readme.md [hunk 3]
```diff
@@ -27,35 +27,31 @@ export default defineComponent({
...
 | Slot      | Description | Data |
 |-----------|-------------|------|
 | _default_ |             | --   |
 
 ## Events
-
-| Event   | Description                      | Data                  |
-|---------|----------------------------------|-----------------------|
-| `input` | Used to update the v-model value | `(string | number)[]` |
+| Event   | Description                      | Data                   | Value |
```
[the remark this answers is not in the document]
Comment (Nitwel) [reply src/components/v-button-group/readme.md:46]: In this case, data should be renamed to value.

**Pull request closed without merging**